index.js

```diff
@@ -394,7 +391,7 @@ const ukLocalAuthorities = [
 {value:"Limavady Borough Council",label:"Limavady Borough Council"},
 {value:"Derry City Council",label:"Derry City Council"},
 {value:"Newry",label:"Newry"},
-{value:" Mourne and Down District Council",label:" Mourne and Down District Council"},
+{value:"Mourne and Down District Council",label:"Mourne and Down District Council"},
```

Comment:
Should this be `Newry, Mourne and Down District Council`?

Reply:
Removed Newry, and ammended to Newry, Mourne and Down District Council.